nixos/modules/system/boot/stage-1.nix

```diff
@@ -30,55 +30,61 @@ let
   fileSystems = filter utils.fsNeededForBoot config.system.build.fileSystems;
 
   # A utility for enumerating the shared-library dependencies of a program
-  findLibs = pkgs.writeShellScriptBin "find-libs" ''
```

Comment:
I think we could do ```buildPackages.writeShellScriptBin``` here

Reply:
Oh hey, that's a much better solution! :-)